csharp/src/Microsoft.ML.OnnxRuntime/OnnxRuntime.cs

```diff
@@ -101,6 +101,34 @@ public void DisableTelemetryEvents()
             NativeApiStatus.VerifySuccess(NativeMethods.OrtDisableTelemetryEvents(Handle));
         }
 
+        /// <summary>
+        /// Queries all the execution providers supported in the native onnxruntime shared library
+        /// </summary>
```

Comment:
I have a project in mind to generate a documentation. Make an effort to be as descriptive as possible.

Reply:
Personally,  I think this specific description is pretty descriptive for this API. Do you have suggestions to make it better ?